<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetsWidget</class>
 <widget class="QWidget" name="SetsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>778</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MySplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QTableWidget" name="sets">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Set</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>21</number>
       </property>
       <item>
        <widget class="QLabel" name="setName">
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Set Name</string>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
            <item>
             <widget class="QTableWidget" name="movies">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Movie</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Sorttitle</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="buttonAddMovie">
                <property name="toolTip">
                 <string>Add movie to set</string>
                </property>
                <property name="text">
                 <string>Add Movie</string>
                </property>
                <property name="dropShadow" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonRemoveMovie">
                <property name="toolTip">
                 <string>Remove selected movie from set</string>
                </property>
                <property name="text">
                 <string>Remove Movie</string>
                </property>
                <property name="dropShadow" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Poster</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyLabel" name="poster">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>300</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Click to Change</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../data/MediaElch.qrc">:/img/placeholders/poster.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="dropShadow" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QToolButton" name="buttonPreviewPoster">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>Full preview</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../data/MediaElch.qrc">
                  <normaloff>:/img/zoom_in.png</normaloff>:/img/zoom_in.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="posterResolution">
                <property name="toolTip">
                 <string>Resolution</string>
                </property>
                <property name="text">
                 <string>Resolution</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Backdrop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyLabel" name="backdrop">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>112</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>112</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Click to Change</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../data/MediaElch.qrc">:/img/placeholders/fanart.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="dropShadow" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QToolButton" name="buttonPreviewBackdrop">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>Full preview</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../data/MediaElch.qrc">
                  <normaloff>:/img/zoom_in.png</normaloff>:/img/zoom_in.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="backdropResolution">
                <property name="toolTip">
                 <string>Resolution</string>
                </property>
                <property name="text">
                 <string>Resolution</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>ui/small_widgets/MyLabel.h</header>
  </customwidget>
  <customwidget>
   <class>MySplitter</class>
   <extends>QSplitter</extends>
   <header>ui/small_widgets/MySplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
